backend/impl/src/main/java/com/virtuslab/gitmachete/backend/impl/hooks/StatusBranchHookExecutor.java

```diff
@@ -31,7 +31,7 @@
 
 @CustomLog
 public final class StatusBranchHookExecutor extends BaseHookExecutor {
-  private static final int EXECUTION_TIMEOUT_SECONDS = 1;
+  private static final int EXECUTION_TIMEOUT_SECONDS = 2;
```

Comment:
Unrel, mostly for the sake of `backend:impl` integration tests passing more reliably